projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -8,3 +8,44 @@ body {
 }
 
 /* ---------------------------------------------------- */
+
+.container {
+	display: inline-flexbox;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: center;
+	background-color: whitesmoke;
+	border-radius: 1rem;
+	align-items: stretch;

```

Comment:
`stretch` je defaultní hodnota, takže když ji neuvedeš, mělo by se to chovat úplně stejně. Můžeš vyzkoušet přes dev tools 😉